frontend/packages/noobaa-storage-plugin/src/selectors/dashboard.ts

```diff
@@ -1,7 +1,6 @@
 import * as _ from 'lodash';
 import { Alert } from '@console/internal/components/monitoring';
 import { PrometheusResponse } from '@console/internal/components/graphs';
-import { K8sResourceKind } from '@console/internal/module/k8s';
 
 export const filterNooBaaAlerts = (alerts: Alert[]): Alert[] =>
```

Comment:
This can also move into utils as this is not a selector

Reply:
right! good one :)